pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -9,19 +9,34 @@ theme: responsive
 
 %h1= I18n.t(:hoc_overview_title)
 
-=I18n.t(:hoc_overview_subtitle)
+= I18n.t(:hoc_overview_subtitle)
 
-#resources.tile-container-responsive
+- if DCDO.get('hoc_launch', nil) == 'dance'
```

Comment:
`CDO.default_hoc_launch` instead of `nil` now

Reply:
good catch!